app/services/ssm_parameters.rb

```diff
@@ -0,0 +1,30 @@
+class SsmParameters
+  PARAMETER_TYPES = %w[String StringList SecureString].freeze
+
+  def initialize(district, name)
+    @district = district
+    @name = name
+  end
+
+  def put_parameter(value, type)
+    unless PARAMETER_TYPES.include?(type)
+      raise ExceptionHandler::InternalServerError.new("Type #{type} is not in #{PARAMETER_TYPES}")
```

Comment:
Internal Server Errors are errors that are unexpected or temporary and we don't want or don't know what to show the client, and it is not the client's fault. In this case, it is the client's fault for not passing the correct parameter type so this should return something like ParameterError or something to tell the client that what it is passing to the server is not correct and cannot be accepted.

Reply:
Maybe we should just only use SecureString, and remove the parameter, which will clean up all the error handling code.

Reply:
@davidsiaw Thank you for the reviews
That makes sense!
I updated the code! 
Can you take a look at when you have time?
https://github.com/degica/barcelona/pull/676/commits/6d7a56b1a809e845bf20ba51c8cc971c5b00117e